src/arduino/debug.h

```diff
@@ -103,14 +103,29 @@
  * Macro pour le message phase + pression
  */
 #ifdef DEBUG
-#define DBG_PHASE_PRESSION(centieme, periode, phase, pression)                                     \
-    if (centieme % periode == 0)                                                                   \
-    {                                                                                              \
-        Serial.print("Phase : ");                                                                  \
-        Serial.print(phase);                                                                       \
-        Serial.print(" - Pression : ");                                                            \
-        Serial.println(pression);                                                                  \
+#define DBG_PHASE_PRESSION(cycle, centieme, periode, phase, subPhase, pression, consigneBlower,  positionBlower, consignePatient, positionPatient)   \
+    if (centieme % periode == 0)                                                                                                  \
+    {                                                                                                                             \        
+        Serial.print(cycle * 300 + centieme);                                                                                                   \
```

Comment:
@jabby 
FYI, the STM32 core provide the printf method for Serial.
So you could do something like this:
```
        Serial.printf("%u;%u;%i;%i;%i;%u;%u;%u;%u;%u\n\n", \
        (cycle * 300 + centieme), cycle, centieme, (300 - phase * 100), (300 - subPhase * 100), pression, consigneBlower, positionBlower, consignePatient, positionPatient); \
```
I did not check all arguments type for the format.